Makefile

```diff
@@ -228,6 +230,14 @@ multiarch-binverify:
 	@echo 'Verifying kube-router gobgp for ARCH=$(FILE_ARCH) ...'
 	@[ `file kube-router gobgp| cut -d, -f2 |grep -cw "$(FILE_ARCH)"` -eq 2 ]
 
+cni-download:
+	@echo Downloading CNI Plugins for $(GOARCH)
+		curl -L -o cni-plugins-$(GOARCH).tgz \
+			https://github.com/containernetworking/plugins/releases/download/$(CNI_VERSION)/cni-plugins-linux-$(GOARCH)-$(CNI_VERSION).tgz
+		mkdir -p cni-download
+		tar -xf cni-plugins-$(GOARCH).tgz -C cni-download
+		rm -f cni-plugins-$(GOARCH).tgz
+
```

Comment:
@mrueg after trying a couple of approaches, this is the way I decided to go. Let me know if you have any thoughts.

Reply:
Yeah that seems fine, if you wanna avoid cURL, you could use ADD with a url as well.

https://docs.docker.com/engine/reference/builder/#add